docs/drivers/spi.md

```diff
@@ -88,7 +88,7 @@ Start an SPI transaction.
 #### Arguments {#api-spi-start-arguments}
 
  - `pin_t slavePin`  
-   The QMK pin to assert as the slave select pin, eg. `B4`.
+   The GPIO pin connected to the desired device's `SS` line.
```

Comment:
I would prefer an update to chip select (cs) as this is used frequently across the repository and drivers.

Reply:
Yeah, I thought about updating all the terminology here as well but I think it probably warrants its own PR for visibility.

Reply:
Alright, let's do it in a separate PR.